solver_rspec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'solver'
+
+describe Solver do
+  context 'Given a new instace of the class' do
+    solver = Solver.new
+
+    it 'test if the method factorial(n) receives an interger parameter and returns (n!)' do
+      expect(solver.factorial(6)).to eq 720
+    end
+
+    it 'test if the method reverse(s) receives a string parameter and returns the reversed string' do
+      expect(solver.reverse('hello')).to eq 'olleh'
+    end
+
+    it 'test if the method fizzbuzz receives an interger parameter and returns a string according to its reminders' do
+      expect(solver.fizzbuzz(15)).to eq 'fizzbuzz'
+      expect(solver.fizzbuzz(3)).to eq 'fizz'
+      expect(solver.fizzbuzz(5)).to eq 'buzz'
+      expect(solver.fizzbuzz(7)).to eq '7'
+    end
+  end
+end
```

Comment:
- [ ] Your tests are not running on my terminal and are giving me 0 examples and 0 failures as seen in the image below stating no examples found. This is because the name of this file should be `solver_spec.rb` and not `solver_rspec.rb` and you should place this file in a `spec` folder.
![image](https://user-images.githubusercontent.com/45460192/175577237-7615672b-2161-43ab-a226-7a414fb3d212.png)